### PR TITLE
Adding mult ace solution

### PR DESCRIPTION
**Pull request by jbieniosek**

### CODE REVIEW

File: tests/test_blackjack_score.py [hunk 1]
```diff
@@ -82,5 +82,22 @@ def test_returns_bust_for_scores_over_21():
...
 
+def test_returns_14_for_ace_ace_ace_ace():
+    # Arrange
+  hand = ['Ace', 'Ace', 'Ace', 'Ace']
+
+  # Act
+  score = blackjack_score(hand)
+
+  # Assert
+  assert score == 14
```
Comment (beccaelenzil) [inline tests/test_blackjack_score.py]: [This PR](https://github.com/Ada-Developers-Academy/core-unit-1/pull/173/files) suggests adding these edge tests as optional for students who completed code to pass the tests above. Here's a suggestion for incorporating the test cases here:  

```suggestion
#*****Consider the following two edge test cases only after you have implemented code to pass all of the tests above*****
@pytest.mark.skip(reason="logic not yet implemented")
def test_returns_12_for_ace_ace_king():
    # Arrange
  hand = ['Ace', 'Ace', 'King']

  # Act
  score = blackjack_score(hand)

  # Assert
  assert score == 12

@pytest.mark.skip(reason="logic not yet implemented")
def test_returns_14_for_ace_ace_ace_ace():
    # Arrange
  hand = ['Ace', 'Ace', 'Ace', 'Ace']

  # Act
  score = blackjack_score(hand)

  # Assert
  assert score == 14
```

Comment (beccaelenzil) [reply tests/test_blackjack_score.py]: If we like this, we should add it to the [tdd-exercise replit](https://replit.com/@adadev/tdd-exercise#main.py)

Comment (jbieniosek) [reply tests/test_blackjack_score.py]: LGTM!

File: tests/test_blackjack_score.py [hunk 2]
```diff
@@ -82,5 +82,25 @@ def test_returns_bust_for_scores_over_21():
   # Assert
   assert score == "Bust"
 
+#*****Consider the following two edge test cases only after you have implemented code to pass all of the tests above*****
+@pytest.mark.skip(reason="logic not yet implemented")
```
Comment (beccaelenzil) [inline tests/test_blackjack_score.py]: I realize now this comment doesn't really make sense in the solution branch, but if we add this test cases to the exercise, we may want to include this comment. Does that make sense? What do you think about adding these test cases to the replit and main branch with this comment?

**Pull request merged**